resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterSchedulerSuite.scala

```diff
@@ -591,18 +591,138 @@ class MesosClusterSchedulerSuite extends SparkFunSuite with LocalSparkContext wi
       new Date())
 
     val expectedCmd = "cd spark-version*;  " +
-        "bin/spark-submit --name \"app name\" --master mesos://mesos://localhost:5050 " +
-        "--driver-cores 1.0 --driver-memory 1000M --class Main " +
-        "--conf \"another.conf=\\\\value\" " +
-        "--conf \"spark.app.name=app name\" " +
-        "--conf spark.executor.uri=s3a://bucket/spark-version.tgz " +
-        "../jar " +
-        "\"--a=\\$2\" " +
-        "--b \"x y z\""
+      "bin/spark-submit --name \"app name\" --master mesos://mesos://localhost:5050 " +
+      "--driver-cores 1.0 --driver-memory 1000M --class Main " +
+      "--conf \"another.conf=\\\\value\" " +
+      "--conf \"spark.app.name=app name\" " +
+      "--conf spark.executor.uri=s3a://bucket/spark-version.tgz " +
+      "../jar " +
+      "\"--a=\\$2\" " +
+      "--b \"x y z\""
```

Comment:
This is a valid indentation fix, but it's irrelevant to `Support for priority queues`. We had better do this clean-up later in another PR in order to keep this PR consistent and neat.

Reply:
Reverted :heavy_check_mark: